### PR TITLE
Project: Weather App

### PR DESCRIPTION
**Pull request by kalavhan**
Pull request for a final review of the project
This pull request has the next functions:

- Search a city by using google places API
- Get weather information from the selected city using Open weather Current API
- Change icon based on weather
- Switch between the imperial and metric system

### CODE REVIEW

File: src/modules/weather-api.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
+const getWeather = () => {
+  async function getWeatherCoor(lat, lon) {
+    const url = `https://api.openweathermap.org/data/2.5/weather?lat=${lat}&lon=${lon}&appid=ff6c534fa692d1c761f8ca2f57394e60`;
+    const response = await fetch(url, {
+      mode: 'cors',
+      credentials: 'omit',
+    });
+    return response.json();
```
Comment (Redvanisation) [inline src/modules/weather-api.js:8]: Wrap this in a **try** block and add a **catch** block to handle the errors after it.

File: src/modules/weather-api.js [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+  }
+
+  async function getWeatherCity(city, state, country) {
+    const url = `https://api.openweathermap.org/data/2.5/weather?q=${city},${state},${country}&appid=ff6c534fa692d1c761f8ca2f57394e60`;
+    const response = await fetch(url, {
+      mode: 'cors',
+      credentials: 'omit',
+    });
+    return response.json();
```
Comment (Redvanisation) [inline src/modules/weather-api.js:17]: Same this as the previous function.

**Comment by kalavhan** [comment]
Hello @Redvanisation, thanks a lot for the review! I have wrapped the await inside a try-catch and I added the dev script to the package.json
Have a nice day!

**Comment by kalavhan** [comment]
Hello @bellom, Thanks a lot!
Have a wonderful day!

**Pull request merged**